docs/conf.py

```diff
@@ -30,19 +30,14 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
-    """Do not warn on external images."""
-    if not msg.startswith('nonlocal image URI found:'):
-        self._warnfunc(msg, '{0}:{1}'.format(get_source_line(node)))
-
-sphinx.environment.BuildEnvironment.warn_node = _warn_node
-
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# Do not warn on external images.
```

Comment:
@tiborsimko … or in the situation where a dependent package declare ``Sphinx>=1.5`` and we have ``Sphinx>=1.3,<1.5`` (i.e. a version conflict) … no matter if we used open-ended or locked requirements we will eventually have conflicts one way or another. Only way to avoid conflicts is by doing a ``pip freeze > requirements.txt`` (which could be done for e.g. the demo site).